src/main/resources/swagger/api-docs.yaml

```diff
@@ -2731,6 +2731,54 @@ paths:
         200:
           description: OK
 
+  '/api/notifications/workspace/{workspaceNamespace}/{workspaceName}':
+    get:
+      responses:
+        200:
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/NotificationType'
+      parameters:
+        - in: path
+          description: workspace namespace
+          name: workspaceNamespace
+          required: true
+          type: string
+        - in: path
+          description: workspace name
+          name: workspaceName
+          required: true
+          type: string
+      tags:
+        - notifications
```

Comment:
capitalize this tag and the tag on the endpoint below

(I hate to nitpick on something this small but this is the user-facing swagger so I think it's worth being consistent with the style of the other tags)